UPGRADE-3.4.md

```diff
@@ -68,6 +68,9 @@ Process
 
  * The `Symfony\Component\Process\ProcessBuilder` class has been deprecated,
    use the `Symfony\Component\Process\Process` class directly instead.
+   
+ * The `Symfony\Component\Process\Process` __construct `cwd` argument will throw 
+   a deprecation warning on `run` if the folder does not exist.
```

Comment:
we should replace the (silent) deprecation log by a real exception in 4.0, `UPGRADE-4.0.md` should mention that

Reply:
I believe this has been done properly now.